tests/installation/ipxe_install.pm

```diff
@@ -170,12 +168,13 @@ sub run {
             record_info("TW install?", "Pls make sure it is Tumbleweed that is being installed.", result => 'softfail');
         }
         assert_screen([qw(network-config-created loading-installation-system)], 60);
+        return if get_var('AUTOYAST');
         wait_still_screen(stilltime => 12, similarity_level => 60, timeout => 30) unless check_screen('sshd-server-started', timeout => 60);
         save_screenshot;
     }
 
     # when we don't use autoyast, we need to also load the right test modules to perform the remote installation
-    if (get_var('AUTOYAST') and !get_var('VIRT_AUTOTEST')) {
+    if (get_var('AUTOYAST')) {
         # make sure to wait for a while befor changing the boot device again, in order to not change it too early
         sleep 120;
     }
```

Comment:
Just a reminder, with so many if else to differentiate the handling for tumbleweed, the readability of this file is decreasing. Maybe it is good to consider if another way is an option for future similar case, namely `inheriting from ipxe_install, then rewrite sub run`

Reply:
This PR is just to stablize the TW tests, so to optimizing test structure is not considered, but it can be included in future test.